capsules/src/tickv.rs

```diff
@@ -97,7 +97,7 @@ impl<'a, F: Flash> tickv::flash_controller::FlashController<64> for TickFSFlastC
 
         if self
             .flash
-            .write_page((0x20040000 + address) / 64, data_buf)
+            .write_page((0x20060000 + address) / 64, data_buf)
```

Comment:
I know this isn't relevant to this PR, but why does the tickv capsule have a hardcoded address?

Reply:
Yeah, it needs to be removed. I only just noticed it as well. That should be a separate PR though